app/controllers/notification_logs_controller.rb

```diff
@@ -0,0 +1,23 @@
+class NotificationLogsController < ApplicationController
+  def create
+    NotificationLog.create(notification_log_params)
+
+    respond_to do |format|
+      format.json { render json: {message: "Notification Log created"}, status: 202 }
+    end
+  end
+
+private
+
+  def notification_log_params
+    params.permit(
+      :gov_request_id,
+      :content_id,
+      :public_updated_at,
+      :document_type,
+      :app,
```

Comment:
Is there a better name than just app for this? Like `email_notification_app` or something?